### DISCUSSION TITLE
To choose "Accept Solution" without clicking.(short cut)

### DISCUSSION THREAD

**Discussion by Yuya-Maeda** [community/community#7256] [Copilot Conversations]
**About using 4.Getting more suggestions**

After Ctrl+Enter, and getting more suggestions, if there is way to choose "Accept Solution" without clicking, that would be better (without moving your hand away from the keyboard.)

I'm a Mac User and use Emac commands to cycling through suggestion, (ctrl +n for go up  and ctrl +p for go down).
If there is a way to select "Accept solution",  it'll be life saver.

**Comment by jansorg** [comment]
That's in a JetBrains IDE, right?
In that case you could use Tab and Shift+Tab to focus the different "Accept solution" buttons.
Or are you looking for a way to change these shortcuts?

**Reply by Yuya-Maeda** [reply]
Thank you for the reply.

Sorry I should have mentioned. It's in a VS code on Mac.
In VS Code, Tab and Shift+Tab just navigate to other tab window or buttons (like split buttons and so on , i hope you know what i mean).

I was looking for a way to focus "accept solution" to select and insert a suggestion with hitting Enter key without using mouse or trackpad.
